docs-ref-conceptual/includes/az-upgrade.md

```diff
@@ -11,4 +11,4 @@ Beginning with version [2.11.0](/cli/azure/release-notes-azure-cli#august-25-202
 az upgrade
 ```
 
-This command also updates all installed extensions by default. For more `az upgrade` options, see the [command reference page](/cli/azure/reference-index#az_upgrade).  For Azure CLI versions prior to [2.11.0](/cli/azure/release-notes-azure-cli#august-25-2020), update by reinstalling as described in [Install the Azure CLI](../install-azure-cli.md).
+This command also updates all installed extensions by default. For more `az upgrade` options, see the [command reference page](/cli/azure/reference-index#az_upgrade).  For Azure CLI versions prior to [2.11.0](/cli/azure/release-notes-azure-cli#august-25-2020), update by reinstalling as described in [Install the Azure CLI](../install-azure-cli.md).
```

Comment:
What is changed?

Reply:
Newline at the end of file is removed accidentally. 😂